Tests/Editor/AvatarLoaderTests.cs

```diff
@@ -173,5 +180,37 @@ public IEnumerator AvatarLoader_Cancel_Loading()
             Assert.AreNotEqual(FailureType.None, failureType);
             Assert.AreEqual(null, avatar);
         }
+
+        [UnityTest, CanBeNull]
```

Comment:
Also forgot to mention, why was this CanBeNull macro added? 

I believe it's not necessary, and can be removed along with the 
```cs
using JetBrains.Annotations;
```
At the top